### ISSUE TITLE
Can't debug when following the instructions

### ISSUE THREAD

**Issue by StingyJack** [Microsoft/vscode#47048]
I followed the directions [here](https://github.com/Microsoft/vscode/wiki/How-to-Contribute#build-and-run-from-source) and after waiting a day was able to get `yarn` to run. 

I continued the directions at [Run](https://github.com/Microsoft/vscode/wiki/How-to-Contribute#run), by running the bat file as administrator (gave an error when not running as admin). 

Here's where it starts to lose me (only a little bit)
> You can identify the development version of Code by the Electron icon in the Dock or Taskbar.

I dont have any idea what the Electron icon is or why I should be seeing it when I'm trying to run/debug VS Code.  (I know what electron is)
![image](https://user-images.githubusercontent.com/15381181/38169654-e1756794-353d-11e8-853a-1980af831b87.png)
I dont have it in the system tray regardless.  Not sure what the "dock" is that is referred to. 

Then I get properly stuck  The next step says... 
> Install the Debugger for Chrome extension. This extension will let you attach to and debug client side code running in Chrome. 

But I cant install extensions. 
![image](https://user-images.githubusercontent.com/15381181/38169639-a2046e20-353d-11e8-8995-7a2e1b630fea.png)

Installing the extension in an installed version of vs code is possible, and has been done, but that is not what the instructions say, and I have no idea what to do at this point - how to set a breakpoint, attach to which process, etc.  Is my system missing something, or are the directions omitting a step?

**Comment by StingyJack** [comment]
Yes, i have that icon in the taskbar, "dock" had me looking in the system tray (to the right of the taskbar, by the clock). Adding that picture to those instructions would probably be good.

I have experience with extension development with VS. In that model you open VS, set a debug option to launch or attach to an instance of VS (usually with the experimental switch) and debug. This is usually done with the same version, so thats the pattern I tried to follow first. 

Since logging this one of the things I tried was using an installed VSC, opening the folder where the vscode repo was cloned to on my machine, and then I was able to use the launch option.  Is this the preferred or correct way to debug?

I'm trying to figure out why I have to [rename powershell script extensions to get VSC to recognize them  as debug targets](https://github.com/PowerShell/vscode-powershell/issues/641#issuecomment-377521354). 

> If you run out of source, you don't have access to the Marketplace.  

Where can I get more source? Oh...

**Comment by weinand** [comment]
- VS Code is not VS. Do not try to apply anything that you know from VS to VS Code.
- For extension development in VS Code you do not have to build VS Code and you don't need the source of VS Code. Just follow these simple steps: https://code.visualstudio.com/docs/extensions/example-hello-world#_generate-a-new-extension
- I cannot follow your train of thoughts with your statement:
  > I'm trying to figure out why I have to rename powershell script extensions to get VSC to recognize them as debug targets.

because this is for debugging PowerShell scripts.


What are your trying to do?
- develop an extension for VS Code (and debugging the extension)
- extending the core of VS Code (and debugging both the UI and the other involved processes)
- debugging PowerShell

**Comment by StingyJack** [comment]
Frankly at this point, I dont really care. I already deal with one broken Visual Studio (2017) every day, I don't really need the headache of another. Code _looks_ promising, but its only been solid so far as a mediocre markdown editor.

**Comment by weinand** [comment]
From your last comments I'm pretty sure that your problem is not VS Code but the PowerShell extension. VS Code has no built-in PowerShell support. So this is the wrong repository.

**Comment by StingyJack** [comment]
Is the right repository if I wanted to debug the editor, which I did at one point, to find out what its passing to the extension.
